test/integration/full/isolated-env/isolated-env.js

```diff
@@ -40,11 +44,16 @@ describe('isolated-env test', function () {
     });
 
     var isloadedPromise = new Promise(function (resolve, reject) {
-      window.addEventListener('message', function (msg) {
-        if (msg.data === 'axe-loaded') {
-          resolve();
-        }
-      });
+      if (messages.includes('axe-loaded')) {
+        resolve();
+      } else {
+        window.addEventListener('message', function (msg) {
```

Comment:
Minor thing and since this in our tests I don't think it'll matter _too_ much. But should we not remove this listener after it has resolved? That way there are no extraneous listeners during test execution.